net-misc/youtube-viewer/youtube-viewer-3.5.8-r1.ebuild

```diff
@@ -35,7 +35,7 @@ RDEPEND="
 		virtual/freedesktop-icon-theme
 		x11-libs/gdk-pixbuf:2[X,jpeg]
 	)
-	|| ( media-video/ffmpeg[openssl] media-video/ffmpeg[gnutls] )
+	|| ( media-video/ffmpeg[openssl,-libressl] media-video/ffmpeg[-openssl,libressl] media-video/ffmpeg[gnutls] )
```

Comment:
Okay, umm. This isnt wrong, but theres a better way to do this: 
See https://wiki.gentoo.org/wiki/Project:LibreSSL
```
ssl? (
    gnutls? ( net-libs/gnutls )
    !gnutls? (
        !libressl? ( dev-libs/openssl:0= )
        libressl? ( dev-libs/libressl:0= )
    )
)
```
this is the standard way of dealing with situation like this in Gentoo. Could you implement it in your ebuild please?